### PR TITLE
Feature/v2 events

### PR DESCRIPTION
**Pull request by sequoiaat**

### CODE REVIEW

**Comment by aaaronlopez** [comment]
@sequoiaat looks good, I'd just ask we expose a public enum dictionary with all of the BRANCH_STANDARD_EVENTS to make it easy to implement all of the STANDARD_EVENTS similar to what we have for react native: https://github.com/BranchMetrics/react-native-branch-deep-linking/blob/master/src/BranchEvent.js#L196-L294

cc @Sarkar @GeneShay

**Comment by aaaronlopez** [comment]
one more thing, if possible can we add a "DEPRECATED" note to `sendCommerceEvent` and `userCompletedAction`?

**Comment by sequoiaat** [comment]
@aaaronlopez We exposed enum for all standard events.

**Comment by sequoiaat** [comment]
@aaaronlopez put the comment "DEPRECATED" for both methods

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
 var exec = require("cordova/exec");
-var deviceVendor = (typeof window.clientInformation != 'undefined' && typeof window.clientInformation.vendor != 'undefined') ? window.clientInformation.vendor : "unknownVendor";
-
+var deviceVendor = window.clientInformation.vendor;
```
Comment (aaaronlopez) [inline src/index.js]: @sequoiaat why remove the check for `window.clientInformation='undefined'`?

**Comment by aaaronlopez** [comment]
👍

**Pull request merged**